frontend/packages/vjs-plugin/src/styles.scss

```
@@ -1725,17 +1725,17 @@ $controlBarAnimationsDuration: 0.1s;
 			font-stretch: 100%;
 			text-decoration: none;
-			text-shadow: 0 0 2px rgba(#000, 0.5);
+			text-shadow: 0 0 2px rgba(var(--cinemata-black), 0.5);
 			@include multiline_texts_excerpt(
 				$font-size: 18px,
 				$line-height: 23.4px,
 				$lines-to-show: 2,
 				$bg-color: transparent
 			);
-			color: #eee;
+			color: var(--site-featured-bg-color);
```

Stylelint (17.11.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)